src/core/lib/iomgr/ev_epollex_linux.cc

```diff
@@ -135,17 +136,13 @@ static char* pollable_desc(pollable* p) {
 static pollable* g_empty_pollable;
 
 static grpc_error* pollable_create(pollable_type type, pollable** p);
-#ifdef NDEBUG
-static pollable* pollable_ref(pollable* p);
-static void pollable_unref(pollable* p);
-#define POLLABLE_REF(p, r) pollable_ref(p)
-#define POLLABLE_UNREF(p, r) pollable_unref(p)
-#else
-static pollable* pollable_ref(pollable* p, int line, const char* reason);
-static void pollable_unref(pollable* p, int line, const char* reason);
-#define POLLABLE_REF(p, r) pollable_ref((p), __LINE__, (r))
-#define POLLABLE_UNREF(p, r) pollable_unref((p), __LINE__, (r))
-#endif
+static pollable* pollable_ref(pollable* p,
```

Comment:
nit: do you need the forward decl here? would you be able to move them (and the macros) down next to the function?

Reply:
moved up instead